examples/app.js

```diff
@@ -1,123 +1,55 @@
+'use strict';
+
 const express = require('express');
+const _ = require('lodash');
+
 const app = express();
+app.use(express.json());
 const port = 3000;
 
+// Mocks
+const searchResultMock = require('./mocks/searchResult.json');
+const traceResultMock = require('./mocks/traceResult.json');
+const traceMock = require('./mocks/trace.json');
+
+// Static directories
 const path = require('path');
 app.use('/', express.static(path.join(__dirname, '/')));
 app.use('/src', express.static(path.join(__dirname, '../src')));
+app.use('/node_modules', express.static(path.join(__dirname, '../node_modules')));
```

Comment:
Why do we need node modules? The node modules while most likely not be available everywhere?

Reply:
This is the way I expose the lodash dependency for the example html file